ivy/functional/backends/paddle/activations.py

```diff
@@ -68,18 +68,13 @@ def gelu(
     approximate: bool = False,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
+    if x.dtype in [paddle.complex64, paddle.complex128]:
+        return (
+            0.5
+            * x
+            * (1 + paddle_backend.tanh(0.7978845608 * (x + 0.044715 * x * x * x)))
```

Comment:
I know you haven't added this and are just moving it, but ideally we shouldn't use any "magic" numbers in the implementation, probably we can define them as constants to indicate what they signify 😅

Reply:
Unfortunately they don't really signify anything, they're just the numbers that make the approximation work well. I've put a comment in giving the source of them though